pandas/tests/test_groupby.py

```diff
@@ -2481,6 +2481,26 @@ def test_size(self):
             self.assertEqual(result[key], len(group))
 
     def test_count(self):
+        from string import ascii_lowercase
+        n = 1 << 15
+        dr = date_range('2015-08-30', periods=n // 10, freq='T')
+
+        df = DataFrame({
+                '1st':np.random.choice(list(ascii_lowercase), n),
+                '2nd':np.random.randint(0, 5, n),
```

Comment:
add ``dtype='int64'`` for the ints, otherwise these fail on windows (default for ndarray is int32 there)